src/api/web/html/renderHTML.ts

```diff
@@ -98,7 +99,9 @@ export default function renderHTML(config, manifest, manifestFiles, req, res) {
         manifest
       );
       debug('template :: %o', webPage);
-      cache.set('template', webPage);
+      if (needHtmlCache) {
+        cache.set('template', webPage);
+      }
       debug('set template cache');
```

Comment:
We should include debug `debug('set template cache');` within the if condition otherwise is missleading.

Reply:
OK，I have committed it.

Reply:
I've took the liberty to apply some small changes, add suggestion to the config files and just moved your new line to the header of the function close to the other configs.